src/Eventuras.Domain/EventInfoOptions.cs

```diff
@@ -0,0 +1,14 @@
+﻿#nullable enable
+
+namespace Eventuras.Domain;
+
+public class EventInfoOptions
+{
+    public EventInfoRegistrationPolicy RegistrationPolicy { get; set; } = new();
+
+    public class EventInfoRegistrationPolicy
+    {
+        public int? AllowedRegistrationEditHours { get; set; } = null;
+        public bool AllowModificationsAfterCancellationDue { get; set; } = true;
```

Comment:
I think we should set this to false? And possibly it could be named `AllowModificationsAfterLastCancellationDate` to reflect the field name in domain model? 
And possibly extend this PR with a `AllowModificationsBeforeLastCancellationDate` which defaults to true?

Reply:
> which defaults to true

As it is newly introduced business logic / validation, making it false by default will result in breaking change for all existing registrations / events, which might get unexpected for users. 
I don't know do we have rn any user base, but if we do...

Reply:
> could be named `AllowModificationsAfterLastCancellationDate`

I will rename

Reply:
Thanks for your comments @yura-b .. pr now users are not allowed to make any changes to the order after placing it, so a default policy of 

* Users are allowed to edit until LastCancellationDate (default: true) 
* Users are not allowed to edit after LastCancellationDate (default: false)
* Users are allowed to edit registrations and products after LastCancellationDate if it is less <24 since the registration, but it must be before the day the event starts (default: 24)

Would not be a functional breaking change, but will it be a technical breaking change?

Reply:
Fixed